client/rpcserver/handlers.go

```diff
@@ -862,8 +862,10 @@ Registration is complete after the fee transaction has been confirmed.`,
     quote (int): The BIP-44 coin index for the market's quote asset.
     qty (int): The number of units to buy/sell. Must be a multiple of the lot size.
     rate (int): The atoms quote asset to pay/accept per unit base asset. e.g.
-      156000 satoshi/DCR for the DCR(base)_BTC(quote).
-    immediate (bool): Require immediate match. Do not book the order.`,
+	56000 satoshi/DCR for the DCR(base)_BTC(quote).
+    immediate (bool): Require immediate match. Do not book the order.
+    customSwapFeeRate (int): Optional. Custom fee rate to use for swap transactions. The fee
+	may be higher if server's max fee rate is higher, but will never be lower.`,
```

Comment:
There's a little ambiguity here.  The custom rate should be used when `server's match rate < customSwapFeeRate < maxFeeRate`.  This doc line would be clearer if "The fee" were "The actual fee rate used in the swap transaction".  The doc could even say that the setting is used to use a higher fee rate than what might be prescribed by the server.

Reply:
This would also need to specify what asset the rate applies to (it depends on `sell`).